addons/background/src/index.js

```diff
@@ -1,66 +1,7 @@
-import React from 'react';
-import { polyfill } from 'react-lifecycles-compat';
-import PropTypes from 'prop-types';
+import { window } from 'global';
+import ReactBackground from './react';
+import VueBackground from './vue';
 
-import addons from '@storybook/addons';
+const Background = window.STORYBOOK_ENV === 'vue' ? VueBackground : ReactBackground;
```

Comment:
The problem with this approach is that now you bring Vue to react storybook bundle when it's not actually needed. In knobs, we decided to have separate entry points for each framework:

`import { withKnobs } from '@storybook/addon-knobs/react'`

see https://github.com/storybooks/storybook/pull/1832 for reference

Reply:
I see. I will follow this up with a PR to update `@addons/centered` since this is what it is currently doing.

generic framework agnostic addons would awesome!

Reply:
Should I wait for `subscriptionStore` to be merged? or should I just do framework specific exports for now?

Reply:
I think you can add `vue` entry poind without renaming `index` to `react`